packages/cli/yaml/yaml-schema/src/schemas/HttpResponseSchema.ts

```diff
@@ -5,6 +5,7 @@ export const HttpResponseSchema = z.union([
     z.string(),
     WithDocsSchema.extend({
         type: z.string(),
+        responseBodyProperty: z.optional(z.string()),
```

Comment:
Update this should be `returnResponseBodyProperty` (or whatever name we settle on) to match the field defined in the Fern definition.